.circleci/config.yml

```diff
@@ -272,6 +272,11 @@ jobs:
       - image: circleci/php:7.3.14-apache-browsers
     steps:
       - checkout_code
```

Comment:
Looks surplus to requirement but this command does a bunch of other, routine stuff that's handy to keep in. Simpler to just checkout our desired branch after it has completed doing it's stuff.